src/domain/model/episode.go

```diff
@@ -0,0 +1,8 @@
+package model
+
+type Episode struct {
+	Id         int      `json:"id"`
```

Comment:
By convention (most lints even show a warning) the Id fields should be renamed as ID